packages/cli/src/commands/apps/info.ts

```diff
@@ -13,9 +13,7 @@ function formatDate(date: Date) {
 async function getInfo(app: string, client: Command, extended: boolean) {
   const promises = [
     client.heroku.get<Heroku.AddOn[]>(`/apps/${app}/addons`),
-    client.heroku.request<Heroku.App>(`/apps/${app}`, {
-      headers: {Accept: 'application/vnd.heroku+json; version=3.cedar-acm'},
-    }),
+    client.heroku.request<Heroku.App>(`/apps/${app}`),
```

Comment:
Platform API removed `cedar-acm` variant, and it doesn't make sense to pass the standard `Accept` header, that's automatically added by the APIClient if not set.